tests_e2e/scenarios/tests/bvts/run_command.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+# Microsoft Azure Linux Agent
+#
+# Copyright 2018 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# BVT for RunCommand.
+#
+# Note that there are two incarnations of RunCommand (which are actually two different extensions):
+# Microsoft.CPlat.Core.RunCommandHandlerLinux and Microsoft.OSTCExtensions.VMAccessForLinux. This
```

Comment:
Should we change this 
`Microsoft.OSTCExtensions.VMAccessForLinux`
to 
`Microsoft.OSTCExtensions.RunCommand`

Reply:
Yes, will update this, thanks